ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/Client.java

```diff
@@ -280,6 +280,24 @@
    */
   CompletableFuture<ConnectorDescription> describeConnector(String connectorName);
 
+  /**
+   * Define a session variable which can be referenced in sql commands by wrapping it with ${}
+   *
+   * @param variable the name of the variable
+   * @param value the value represented by the variable
+   */
+  void define(String variable, Object value);
+
+  /**
+   * Undefine a variable.
+   */
+  void undefine(String variable);
+
+  /**
+   * @return a map of variables and values
```

Comment:
Clarify that these variables are local session variables for purposes of variable substitution.